include/swift/AST/PrintOptions.h

```diff
@@ -118,6 +118,12 @@ struct ShouldPrintChecker {
   virtual ~ShouldPrintChecker() = default;
 };
 
+enum class PrintInterfaceContentMode : uint8_t {
```

Comment:
Could we use something shorter here? Since it's in a printing file we can probably cut some of the name as already being implied. How about just `ContentLevel` or `InterfaceLevel`?

Reply:
Could rename as InterfaceMode